src/components/Calculator.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+class Calculator extends React.Component {
```

Comment:
* Kindly Use functional components as stated in the [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20functional%20components)